python-formatted-output/person.py

```diff
@@ -0,0 +1,17 @@
+class Person:
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+
+    def __str__(self):
+        return f"I'm {self.name}, and I'm {self.age} years old."
+
+    def __repr__(self):
+        return f"{type(self).__name__}(name='{self.name}', age={self.age})"
+
+
+jane = Person("Jane", 25)
+f"{jane!s}"
+f"{jane!r}"
+"{jane!s}".format(jane=jane)
+"{jane!r}".format(jane=jane)
```

Comment:
Should we add `print()` calls on these to make them show up when running as a script?